js/tweets.js

```diff
@@ -3,80 +3,80 @@
+        Interval();
+        document.getElementsByClassName('tweets-feed')[0].style.opacity =  0;
+        window.setTimeout(parseFunc, 560);
+    }
 }
 
 function parser(data) {
-	var parsed = ""
-	var tweet = data.statuses[tweetNum].text;
-	var words = tweet.split(" ");
-	var loklakLinkCount = 0;
-	for (word in words) {
-		if (words[word].startsWith("@")) {
-			parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("#")) {
-			parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("http")) {
-			if (words[word].startsWith("http://loklak")) {
-				parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
-				loklakLinkCount += 1;
-			} else {
-				parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
-			}
-		} else {
-			parsed += words[word] + " ";
-		}
-	}
-  var date = +new Date(data.statuses[tweetNum].created_at);
-  var myDate = new Date();
-  myDate = myDate.getTime();
-  var difference = myDate-date; //it's in miliseconds
-  var tweetAge = Math.round(difference/1000/3600) > 24 ? Math.round(difference/1000/3600/24) + " days ago" : Math.round(difference/1000/3600) + " hours ago";
-  parsed += "<span class='dateTweeted'>Tweeted "+tweetAge+"</div>";
-
-	document.getElementsByClassName("tweets-feed")[0].innerHTML =  parsed;
-	document.getElementsByClassName("tweets-feed")[0].style.opacity =  1;
+    var parsed = ""
+    var tweet = data.statuses[tweetNum].text;
+    var words = tweet.split(" ");
+    var loklakLinkCount = 0;
+    for (word in words) {
+        if (words[word].startsWith("@")) {
+            parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+        } else if (words[word].startsWith("#")) {
+            parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+        } else if (words[word].startsWith("http")) {
+            if (words[word].startsWith("https://loklak")) {
+                parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
+                loklakLinkCount += 1;
+            } else {
+                parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
+            }
+        } else {
+            parsed += words[word] + " ";
+        }
+    }
+    var date = +new Date(data.statuses[tweetNum].created_at);
+    var myDate = new Date();
+    myDate = myDate.getTime();
+    var difference = myDate-date; //it's in miliseconds
+    difference = Math.round(difference/1000/3600/24);
```

Comment:
I think tweetAge would be better.

Reply:
Ok fixing that too now

Reply:
It shows 0 days, instead if we use tweetAge hours will also be displayed, which is a better option.

![screenshot from 2017-12-20 23-41-22](https://user-images.githubusercontent.com/12656846/34221805-5038fc26-e5df-11e7-9eed-98b3be00225d.png)